pyomo/core/base/indexed_component_slice.py

```diff
@@ -654,8 +759,16 @@ def get_last_index(self):
             return ans
 
     def get_last_index_wildcards(self):
+        """
+        Get a tuple of the values in the wildcard positions for 
+        the most recent indices checked by each _slice_generator
```

Comment:
"checked" is probably the wrong verb.  Wouldn't "returned" be more accurate?

Reply:
The `_slice_generator` returns components, not indices (or at least advancing it does), so I'm averse to "returned." I'll go with "used" for now.

Reply:
Good point.  What about:
```
Return a tuple of the values in the wildcard positions for 
the index corresponding to the component most recently
returned by each _slice_generator.
```